src/Bicep.Cli.IntegrationTests/BuildCommandTests.cs

```diff
@@ -103,7 +103,6 @@ public async Task Build_Valid_SingleFile_WithTemplateSpecReference_ShouldSucceed
         [DataRow("br/mcr:az", true, LanguageConstants.BicepPublicMcrRegistry)]
         [DataRow("br:contoso.azurecr.io/bicep/providers/az", true, "contoso.azurecr.io")]
         // Negative
-        [DataRow("az", false)]
```

Comment:
I am removing this from the negatie because the new syntax allows for an identifier so this becomes a valid case

Reply:
Should you change `shouldSucceed` from `false` to `true` instead of removing the test in that case?

Reply:
I would prefer not to, the subject under test is the logic that resolves provider aliases. LOC 134 interpolates the value in the test case into a provider declaration syntax of the form `provider '{providerDeclarationSyntax}@2.0.0'`. If you apply the values in the `DataRow` attribute this results in `provider 'az@2.0.0'`. This is a legacy syntax that currently is allowed but marked as deprecated, once we deprecate it then the test case should be added again. 
I will resolve this by leaving the test-case commented out with a note to uncomment when the legacy syntax is deprecated.